pages/k8s/1.31/release-notes.md

```diff
@@ -0,0 +1,96 @@
+---
+wrapper_template: templates/docs/markdown.html
+markdown_includes:
+  nav: kubernetes/docs/shared/_side-navigation.md
+context:
+  title: 1.31 Release notes
+  description: Release notes for Charmed Kubernetes
+keywords: kubernetes, release, notes
+tags: 
+  - news
+sidebar: k8smain-sidebar
+permalink: 1.31/release-notes.html
+layout:
+  - base
+  - ubuntu-com
+toc: False
+
+---
+
+# 1.31
+
+### September 04, 2024 - `charmed-kubernetes --channel 1.31/stable`
+
+The release bundle can also be [downloaded here](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.31/bundle.yaml).
+
+## What's new
+### aws-k8s-storage
+* Updates aws-storage manifests to include versions 1.12.1 -> 1.32.0
+### azure-cloud-provider
+* Updates azure resources to include azure-disk versions 1.28.9 -> 1.30.2
+* Updates azure resources to include cloud-provider versions 1.27.14 -> 1.30.0
+### calico
+* Increases charm debug logs to better isolate failures in [LP#2064145](https://bugs.launchpad.net/bugs/2064145)
+### kubernetes-control-plane
+* Improvements to logging when failing to create a cloud-based load-balancer
+* Enhances the upgrade action to avoid clearing charm status warnings
+### openstack-cloud-controller
+* Updates manifests to include up to v1.30.0
+### openstack-integrator
+* Adds project-id to the openstack credentials handed out by this relation
+
+## Component Versions
+
+### Charm/Addons pinned versions
+- kube-ovn 1.12.6
+- calico 3.27.4
+- coredns 1.9.4
+- ingress-nginx 1.6.4
```

Comment:
Wasn't nginx recently updated to `1.11.2` with https://github.com/charmed-kubernetes/charm-kubernetes-worker/pull/178 ?